app/models/organisation.rb

```diff
@@ -74,4 +76,8 @@ def display_attributes
       { name: "data_protection_agreement", value: data_protection_agreement_string, editable: false },
     ]
   end
+
+  def strip_whitespace
+    self.name = name.strip unless name.nil?
```

Comment:
https://api.rubyonrails.org/classes/String.html#method-i-squish might be better than strip?

Reply:
Hmm, could there ever be a reason to have a double space in the name?

Reply:
Not that I know of, but I could be wrong.

Reply:
If we're generalising this, I think I'd prefer just to strip it for now, cause it feels less destructive